### PR TITLE
[Docs] Update Vizro-AI docs to explain how to export the environment to ensure API key visibility

### PR DESCRIPTION
**Pull request by stichbury**
## Description
Answers internal issue 707 (see long discussion thread on Slack) where I was receiving a response of unauthorized to Vizro-AI calls because the environment wasn't exported (despite following set up advice). I've added a section in case it affects others beyond me & @nadijagraca who also reported it.

## Screenshot
You can see the relevant page of the build docs, with the new admonition, here: https://vizro--436.org.readthedocs.build/projects/vizro-ai/en/436/pages/tutorials/quickstart/
## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

  - I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
  - I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorized to submit this contribution on behalf of the original creator(s) or their licensees.
  - I certify that the use of this contribution as authorized by the Apache 2.0 license does not violate the intellectual property rights of anyone else.
  - I have not referenced individuals, products or companies in any commits, directly or indirectly.
  - I have not added data or restricted code in any commits, directly or indirectly.

### CODE REVIEW

**Review by huong-li-nguyen — COMMENTED** [review]
LGTM 👍 But I might not be best to evaluate these doc changes. Could you let @Anna-Xiong  and @nadijagraca know :) 

@l0uden - can you double-check the integration tests here? Seems to be failing due to some chrome installation.

File: vizro-ai/docs/pages/tutorials/quickstart.md [hunk 1]
```diff
@@ -67,6 +67,16 @@ Finally, we call the `plot()` method with our English language instruction, to g
 vizro_ai.plot(df, "create a line graph for GDP per capita since 1950 for each continent. Mark the x axis as Year, y axis as GDP Per Cap and don't include a title")
 ```
 
+!!! warning "Help! My LLM request was unauthorized"
+
+    If you see an error similar to this, your LLM API key is not valid:
+
+    `INFO:httpx:HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 401 Unauthorized"`
+
+    Make sure you have [set up access to a large language model](../user-guides/install/#set-up-access-to-a-large-language-model). If you are confident that you have specified your API key correctly and have sufficient credits associated with it, check your environment. Some developers export the environment explicitly to ensure the API key is available at runtime. Call the following in your terminal:
```
[the remark this answers is not in the document]
Comment (stichbury) [reply vizro-ai/docs/pages/tutorials/quickstart.md:76]: Good idea! I've added a commit to do just that.

**Review by nadijagraca — APPROVED** [review]
Looks good. Thank you for including this. ⭐

**Review by Anna-Xiong — APPROVED** [review]
LGTM! thank you for adding this

**Pull request merged**